pkg/engine/backend/config.go

```diff
@@ -0,0 +1,171 @@
+package backend
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/zclconf/go-cty/cty/gocty"
+
+	workspaceapi "kusionstack.io/kusion/pkg/apis/workspace"
+	backendinit "kusionstack.io/kusion/pkg/engine/backend/init"
+	"kusionstack.io/kusion/pkg/engine/states"
+	"kusionstack.io/kusion/pkg/engine/states/local"
+	"kusionstack.io/kusion/pkg/workspace"
+)
+
+// StateStorageConfig contains backend config for state storage.
+type StateStorageConfig struct {
```

Comment:
Why not name this struct as `StateBackendConfig` since it is under the config pkg?

Reply:
Because it's used to new `StateStorage` which is defined in pkg `engine/states`.